docs/docs/quickstart/add-data-to-google-sheets/README.md

```diff
@@ -0,0 +1,121 @@
+# Insert data into Google Sheets
+
+Next, let's transform data returned by the ISS API and save it to Google Sheets. This example builds on the workflow created in [previous sections](/quickstart/hello-world/) and will cover how to:
+
+[[toc]]
+
+::: tip
+If you didn't complete the previous examples, we recommend you start from the [beginning of this guide](/quickstart/). If you still want to start here, [copy this workflow](https://pipedream.com/@gettingstarted/quickstart-use-any-npm-package-p_pWCg5BP) and then follow the instructions below. If you have any issues completing this example, you can [view, copy and run a completed version](https://pipedream.com/@gettingstarted/quickstart-insert-data-into-google-sheets-p_KwCAR9z).
+:::
+
+### Set up a Google Sheet for this example
+
+First, create a Google Sheet with the columns `Latitude`, `Longitude`, `Timestamp`, and `Date/Time` in the first row:
```

Comment:
Would it make sense to give people a read-only, public template they can copy?

Reply:
good point -- i'll add that.

Reply:
done